README.md

```
@@ -318,13 +377,13 @@ This creates a consolidated table showing all coverage reports:
 ```
 
 **Default behavior** (with emojis):
-| Tests | Skipped | Failures | Errors | Time |
-| ----- | ------- | -------- | ------ | ---- |
+| Tests | Skipped | Failures | Errors   | Time               |
```

markdownlint-cli2 (0.23.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)